ckeditor/ckeditor.js

```diff
@@ -2,7 +2,7 @@
 Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
 For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
 */
-(function(){window.CKEDITOR&&window.CKEDITOR.dom||(window.CKEDITOR||(window.CKEDITOR=function(){var a=/(^|.*[\\\/])ckeditor\.js(?:\?.*|;.*)?$/i,g={timestamp:"J34B",version:"4.11.3",revision:"8b53603e8",rnd:Math.floor(900*Math.random())+100,_:{pending:[],basePathSrcPattern:a},status:"unloaded",basePath:function(){var c=window.CKEDITOR_BASEPATH||"";if(!c)for(var e=document.getElementsByTagName("script"),b=0;b<e.length;b++){var d=e[b].src.match(a);if(d){c=d[1];break}}-1==c.indexOf(":/")&&"//"!=c.slice(0,
+(function(){window.CKEDITOR&&window.CKEDITOR.dom||(window.CKEDITOR||(window.CKEDITOR=function(){var a=/(^|.*[\\\/])ckeditor\.js(?:\?.*|;.*)?$/i,g={timestamp:"J4E8",version:"4.11.3 DEV",revision:"c77aca6fd",rnd:Math.floor(900*Math.random())+100,_:{pending:[],basePathSrcPattern:a},status:"unloaded",basePath:function(){var c=window.CKEDITOR_BASEPATH||"";if(!c)for(var e=document.getElementsByTagName("script"),b=0;b<e.length;b++){var d=e[b].src.match(a);if(d){c=d[1];break}}-1==c.indexOf(":/")&&"//"!=c.slice(0,
```

Comment:
So this SHA (c77aca6fd) is a non-public SHA. Not sure if it is worth trying to stop it from appearing here (probably not, for now).